src/pages/components/date-picker/usage.mdx

```diff
@@ -107,57 +131,326 @@ Date and time pickers allow users to select a single or a range of dates and tim
 
 <Row>
 <Column colLg={8}>
 
-![range, single, and simple date pickers](images/date-picker-usage-1.png)
+![Alignment example](images/date-picker-alignment-01.png)
 
-<Caption>Types of date pickers</Caption>
+</Column>
+</Row>
+
+The calendar itself will always remain a fixed width and is not adjustable. It should always be aligned to the left edge of its assigned text field.
+
+<Row>
+<Column colLg={12}>
+
+![Alignment example](images/date-picker-alignment-02.png)
 
 </Column>
 </Row>
 
 ## Content
 
-#### Labels
+### Main elements
+
+#### Label
+
+- Both date and time pickers must be accompanied with labels.
+- The labels should be clear and descriptive.
+- Range inputs should be being properly labeled with a start and end label.
+
+#### Date format
+
+- The date format can be displayed differently depending on your location. For example, some countries use month/day/year while other use day/month/year.
+- When using a simple date input include the date format in parentheses inline with the label or as helper text below the label.
+- When the using calendar picker the date format will be automated if the user selects from the calendar menu.
+- Only including the date format as placeholder text inside the field is problematic because it will disappear once the user begins typing.
+
+<Row>
+<Column colLg={8}>
+
+![Alignment example](images/date-format-placement.png)
+
+</Column>
+</Row>
+
+#### Time format
+
+- Both the 12-hour and 24-hour time systems are allowed.
+- If using the 12-hour it must be accommodated with an AM/PM selection.
+- Use uppercase letters and no periods for the abbreviations AM and PM.
+- Specific times should specify a timezone.
+
+### Further guidance
+
+For further content guidance, see Carbon’s [content guidelines](https://www.carbondesignsystem.com/guidelines/content/general).
+
+## Universal behaviors
+
+The behaviors listed in this section are universal across all of the types. For behaviors that are unique to each type, see the sections below.
+
+### States
+
+#### Validation
+
+Invalid fields should be clearly marked. In pickers with more than one field, the invalid state should only be set on the individual factor that is triggering the error so the user can clearly understand which to address.
+
+<Row>
+<Column colLg={8}>
+
+![Validation example](images/date-picker-validation.png)
+
+</Column>
+</Row>
+
+### Localization
+
+Localization, sometimes referred to as internationalization, refers to software adapting to different languages, regional peculiarities and technical requirements of a target locale without additional code changes. This means that if the location is known then formatting of a date or time can automatically change to the acceptable local format. You should always try and design for localization.
```

Comment:
```suggestion
Internationalization, also referred to as globalization, refers to software adapting to different languages, regional peculiarities, and technical requirements of a target locale without additional code changes. This means that if the location is known, then formatting of a date or time can automatically change to the acceptable local format. You should always try to design for internationalization.
```

Reply:
@janchild So we're not using the term "localized" at all? I think I use it a couple of times elsewhere in the documentation. Should I change it to "internationalized" every where?

Reply:
Never mind, read your initial comment, got it.

Reply:
@aagonzales, let's check with @joshblack to make sure we're aligned on terminology. 

Josh, are you using this term also? I believe we could use internationalized or globalized in this kind of situation. 

For context, here is my earlier note:
I based the edit in the internationalization/globalization section on the definitions here. https://www.w3.org/International/questions/qa-i18n. The differences are subtle. From what is explained here, internationalization/globalization is the design and development to prep for ease of localization, which is the adaptation that comes after the product has been built. I'd be fine with using the word globalization instead of internationalization if you'd prefer.